modules/user/inputs.tf

```
@@ -0,0 +1,35 @@
+variable "user" {
+  default     = null
+  description = "The user to create"
+  type = object({
+    name   = string
+    path   = string
+    tags   = map(any)
+    groups = list(string) # group names
+  })
+  nullable = true
```

```suggestion
  nullable = true
  deprecated = "Use the top level inputs (`name`, `patch`, `tags`, `groups` instead of this object."
```

Will make a little noise in the TFC console, but makes the intention clearer and more likely to be fixed later.